src/main/java/frc/robot/Robot.java

```diff
@@ -96,6 +113,16 @@ public void teleopPeriodic() {
         } else {
             mIntakeMotor.set(ControlMode.PercentOutput, 0);
         }
+        if (m_XboxController.getAButton()) {
+            mSolenoidOne.set(DoubleSolenoid.Value.kForward);
+            mSolenoidTwo.set(DoubleSolenoid.Value.kForward);
+        } else if (m_XboxController.getBButton()) {
+            mSolenoidOne.set(DoubleSolenoid.Value.kReverse);
+            mSolenoidTwo.set(DoubleSolenoid.Value.kReverse);
+        } else {
+            mSolenoidOne.get();
```

Comment:
Same thing, get doesn't do anything. I would recommend that you remove this part.